### PR TITLE
Refactor to better support WalletLink based web3 provider

### PR DESCRIPTION
**Pull request by dpereskokov**
WalletLink codebase is pulled from the fork. This way it will be easy to pull and merge new changes from original repo. To build project you have to type: 

`make install`

To Test: 
- [ ] https://migrate.makerdao.com
- [ ] https://etherscan.io/address/0xf5dce57282a584d2746faf1593d3121fcac444dc#writeContract
- [ ] https://app.ens.manager
- [ ] https://kyberswap.com/swap
- [ ] https://compound.finance/
- [ ] https://peepeth.com/welcome
- [ ] https://mycrypto.com/
- [ ] https://www.cryptokitties.co/
- [ ] https://js-eth-sign-2.surge.sh

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+echo '\ngetting Trust WalletLink...'
+if [ -d src ]; then
+    cd  src
+
+    git pull git@github.com:dpereskokov/walletlink.git
```
Comment (dpereskokov) [inline install.sh]: @vikmeup @hewigovens may be it will be better to have this fork inside Trust github?

**Comment by dpereskokov** [comment]
Also it is needed to adopt CI to the new build pipeline

File: install.sh (hunk 1 above)
Comment (hewigovens) [reply install.sh]: yes, we can move it to Trust organization, why not submodule? custom shell script doesn't look like a normal js/npm project

**Comment by hewigovens** [comment]
let's disable CI here?

File: Example/Podfile [hunk 2]
```diff
@@ -3,7 +3,7 @@ platform :ios, '11.0'
 
 target 'TrustWeb3Provider_Example' do
   pod 'TrustWeb3Provider', :path => '../'
-  #pod 'TrustWalletCore'
+  pod 'TrustWalletCore'
```
Comment (vikmeup) [inline Example/Podfile]: can we just include models for Wallet Core?

Comment (hewigovens) [reply Example/Podfile]: This is the example app to test, not the podspec itself

Comment (vikmeup) [reply Example/Podfile]: correct, would it be possible to try just models?

Comment (hewigovens) [reply Example/Podfile]: no, we need to test signing

Comment (hewigovens) [reply Example/Podfile]: https://github.com/trustwallet/trust-web3-provider/blob/731666378ff9ca32c2729f1892c07795c6de6aa4/TrustWeb3Provider.podspec doesn't depend on wallet core

Comment (vikmeup) [reply Example/Podfile]: got it, I was thinking to try test models pod :) looks good then

**Pull request closed without merging**